experiments/AMIP/modular/components/land/bucket_utils.jl

```diff
@@ -1,8 +1,8 @@
 """
     get_land_temp(slab_sim::BucketSimulation)
 
-Returns the surface temperature of the earth; 
-a method for the bucket model 
+Returns the surface temperature of the earth;
+a method for the bucket model
 when used as the land model.
 """
 function get_land_temp(slab_sim::BucketSimulation)
```

Comment:
If you want, you could also use this: ClimaLSM.surface_temperature
https://github.com/CliMA/ClimaLSM.jl/blob/main/src/Bucket/bucket_parameterizations.jl#L36